tsconfig.json

```diff
@@ -0,0 +1,105 @@
+{
+  "include": ["cli/**/*", "generators/**/*", "jdl/**/*", "lib/**/*", "test/**/*"],
+  "exclude": ["**/*.spec.*"],
+  "compilerOptions": {
+    /* Visit https://aka.ms/tsconfig to read more about this file */
```

Comment:
I think we can remove commented configs.

Reply:
Done